### PR TITLE
fix: rendereable incongruence

### PR DESCRIPTION
**Pull request by AjimenezDCL**
`AssetPromise` for GLTF and AB_GameObjects were not updating `asset.renderers` on certain cases when using the promise with a parent set.

This would end up in incongruences between `asset.container.GetComponentsInChildren<Renderer>()` and `asset.renderers`.

### CODE REVIEW

**Review by BrianAmadori — APPROVED** [review]
Can we please add a single test to validate this fix?

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/Utils/Utils.cs [hunk 1]
```diff
@@ -530,5 +527,12 @@ public static void SetLayerRecursively(Transform transform, int layer)
         /// <param name="volume">Linear volume (0 to 1)</param>
         /// <returns>Value for audio mixer group volume</returns>
         public static float ToAudioMixerGroupVolume(float volume) { return (ToVolumeCurve(volume) * 80f) - 80f; }
+
+        public static string GetHierarchyPath(this Transform transform)
```
Comment (BrianAmadori) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/Utils/Utils.cs:531]: If this method isn't used we should remove it?

**Review by BrianAmadori — APPROVED** [review]
LGTM!

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/Utils/Utils.cs (hunk 1 above)
Comment (AjimenezDCL) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/Utils/Utils.cs:531]: Oops I bring this util from the Avatar Refactor for debugging and I forgot to remove it. On it

**Pull request merged**